spec/features/nav_spec.rb

```diff
@@ -2,66 +2,76 @@
 
 feature 'Main Navigation' do
 
-  # TODO: fix failing specs
+  scenario 'should highlight first and second level in header', js: true do
```

Comment:
I added some similar tests in the nav PR I have open (that I'm failing to update). Let's get that merged and see where these overlap.